models/UserCampaign.js

```diff
@@ -0,0 +1,20 @@
+const Model = require('./BaseModel')
+const { ref } = require('objection')
+
+class UserCampaign extends Model {
+  static get tableName () {
+    return 'users_campaigns'
+  }
+
+  static async getUserCountByMotive (motive) {
+    const result = await UserCampaign.query()
```

Comment:
@duranmla you should do the count in SQL instead of JavaScript. Let's change this statement. Also, instead of doing this per motive, why we just don't return the totals for each motive in one query?